src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -20,7 +20,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
-  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(GeneratePlatformNotSupportedAssembly)' != 'true'">
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(GeneratePlatformNotSupportedAssemblyMessage)' == ''">
```

Comment:
It's a bit wacky to condition including source files based on a message. I suggest changing this condition and the one below to

'<ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' and '$(TargetGroup)' != 'netstandard'">'
which is clearer.